includes/Stories_Renderer/Renderer.php

```diff
@@ -521,7 +533,7 @@ protected function get_content_overlay() {
 		}
 
 		?>
-		<div class="story-content-overlay web-stories-list__story-content-overlay">
+		<div class="story-content-overlay">
```

Comment:
I'd prefer having a `web-stories` prefix here. `story-content-overlay` is quite generic otherwise.

`web-stories-list__story-content-overlay` seems alright, doesn't it?

Reply:
yep, it does, this was an unnecessary change, I'll update.